### ISSUE TITLE
A11y_code.visualstudio.com_API Working With Extensions_Continuous Integration_Keyboard: Keyboard focus is not clearly visible on "Dismiss update button".

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#3706]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS2.4.7;#Win10;#Visual Studio Code;#Keyboard; #CAITestPass_May2020;#FTP;#STP;

**Environment Details**:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps**:
1. Launch the application URL :https://code.visualstudio.com
2. Navigate to "API" link.
3. From left navigation, select "Working With Extensions".
4. Navigate to "Continuous Integration" link.
5. Navigate to "Dismiss update button" using tab key.

**Actual**:
Keyboard focus is not clearly visible on "Dismiss update button".

**Similar issue observed in through out the application where "Dismiss update button" is present.**

**Expected**:
Keyboard focus should be clearly visible on on all actionable controls..

**Attachment link for Reference**:
![3706_A11y_code visualstudio com_API Working With Extensions_Continuous Integration_Keyboard Keyboard focus is not clearly visible on Dismiss update button](https://user-images.githubusercontent.com/52035395/83004564-71b6cf80-a02d-11ea-9b7b-3a9c3dad1e51.gif)

**Comment by JacksonKearl** [comment]
Now: 
![image](https://user-images.githubusercontent.com/8586769/87467541-f73a1300-c5cc-11ea-89a1-4f4b4eeabccb.png)

**Comment by Rama8345** [comment]
Verified the issue at https://code.visualstudio.com/api/working-with-extensions/continuous-integration. The issue is fixed. So closing the bug. Please find below attachment for reference. Thanks
**Github Tags:**#Closed;
![3706_Closed](https://user-images.githubusercontent.com/115216656/219320390-8707a9f3-0a04-442b-84d5-ab2a6b754a5f.gif)
